packages/sanity/package.json

```diff
@@ -262,6 +262,7 @@
     "tar-fs": "^2.1.1",
     "tar-stream": "^3.1.7",
     "use-device-pixel-ratio": "^1.1.0",
+    "use-effect-event": "^1.0.2",
```

Comment:
This is already part of other packages installed with sanity, like `@sanity/ui` , `portable-text` etc.

```
➜  sanity git:(corel-history) ✗ pnpm why use-effect-event
Legend: production dependency, optional only, dev only

sanity-root@0.0.0 /Users/pedrobonamin/code/sanity

devDependencies:
@sanity/tsdoc 1.0.134
└─┬ @sanity/ui 2.10.0-corel.0
  └── use-effect-event 1.0.2
@sanity/ui 2.10.0-corel.0
└── use-effect-event 1.0.2
sanity link:packages/sanity
├─┬ @portabletext/editor 1.11.3
│ └── use-effect-event 1.0.2
├─┬ @sanity/insert-menu 1.0.11-release.4
│ └─┬ @sanity/ui 2.10.0-corel.0
│   └── use-effect-event 1.0.2
├─┬ @sanity/presentation 1.17.8-release.4
│ ├─┬ @sanity/ui 2.10.0-corel.0
│ │ └── use-effect-event 1.0.2
│ └── use-effect-event 1.0.2
├─┬ @sanity/ui 2.10.0-corel.0
│ └── use-effect-event 1.0.2
├─┬ react-rx 4.1.7
│ └── use-effect-event 1.0.2
└── use-effect-event 1.0.2

```